pkgs/development/libraries/xdg-dbus-proxy/default.nix

```diff
@@ -9,9 +17,20 @@ stdenv.mkDerivation rec {
     sha256 = "03sj1h0c2l08xa8phw013fnxr4fgav7l2mkjhzf9xk3dykwxcj8p";
   };
 
-  nativeBuildInputs = [ pkgconfig libxslt docbook_xsl docbook_xml_dtd_43 ];
-  buildInputs = [ glib ];
-  checkInputs = [ dbus ];
+  nativeBuildInputs = [
+    pkgconfig
+    libxslt
+    docbook_xsl
+    docbook_xml_dtd_43
+  ];
```

Comment:
I'm noticing this isn't even a libarary? The output is just a single executable `xdg-dbus-proxy`, and we're missing `wrapGAppsHook`. This also concerns me if it's needed in PATH and it's not there.

Reply:
Yup, it is simple security utility. I do not see why it would require `wrapGAppsHook`.